<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubDlg_SearchObj</class>
 <widget class="QDialog" name="SubDlg_SearchObj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Criteria</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,3">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="3,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_value"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Press F3 for Find Next, and F2 for Find Previous.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_casesensitive">
       <property name="text">
        <string>Case Sensitive</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Press Ctrl + F to open this window.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Search By</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QRadioButton" name="radioButton_id">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_defname">
            <property name="text">
             <string>Defname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_description">
            <property name="text">
             <string>Description</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_value</tabstop>
  <tabstop>checkBox_casesensitive</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>radioButton_id</tabstop>
  <tabstop>radioButton_defname</tabstop>
  <tabstop>radioButton_description</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
